pandas/tests/test_window.py

```diff
@@ -275,6 +275,11 @@ def test_how_compat(self):
                             getattr(s, t)(freq='D', **kwargs), op)(how=how)
                         assert_series_equal(result, expected)
 
+    # GH #12373 : rolling functions error on float32 data
+    def test_ensure_type_float64(self):
+        series_float32 = pd.Series(np.arange(5, dtype=np.float32))
```

Comment:
here is a list of types: http://docs.scipy.org/doc/numpy-1.10.1/user/basics.types.html .  Probably can skip the complex and bool types.

Should we also add tests for the other functions like `min()`, `std()`, `median()`, etc.?